src/main/java/org/prebid/server/bidder/conversant/ConversantBidder.java

```diff
@@ -138,14 +139,26 @@ private static Imp modifyImp(Imp imp, ExtImpConversant impExt) {
         return imp.toBuilder()
                 .displaymanager(DISPLAY_MANAGER)
                 .displaymanagerver(DISPLAY_MANAGER_VER)
-                .bidfloor(impExt.getBidfloor())
+                .bidfloor(getBidFloor(imp, impExt))
                 .tagid(impExt.getTagId())
                 .secure(getSecure(imp, impExt))
                 .banner(modifyBanner(banner, impExt.getPosition()))
                 .video(video != null && banner == null ? modifyVideo(video, impExt) : video)
                 .build();
     }
 
+    private static BigDecimal getBidFloor(Imp imp, ExtImpConversant impExt) {
```

Comment:
We don't need whole imp and impExt, just `BigDecimal impBidFloor, BigDecimal impExtBidFloor` will be enough